http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -1599,6 +1599,13 @@ paths:
           required: true
           schema:
             type: string
+        - in: query
```

Comment:
We probably want to have `accessScope` in the `CreateAppRequest` object in the request body, instead of the query string.

Reply:
What do you mean by that? The swagger doc points towards the `query` option https://swagger.io/docs/specification/describing-parameters/